src/App.js

```diff
@@ -4,14 +4,16 @@ import Reviews from "./components/Reviews";
 import { Routes, Route } from "react-router-dom";
 import Header from "./components/Header";
 import Nav from "./components/Nav";
+
 class App extends Component {
   render() {
     return (
       <div className="App">
         <Header />
         <Nav />
         <Routes>
-          <Route path="Reviews" element={<Reviews />}>r</Route>
+          <Route path="Reviews" element={<Reviews />}></Route>
```

Comment:
Watch your casing and paths. 
should be lowercase and start with a `/` eg `/reviews`